internal/infraestructure/github-connection/github-connection.go

```diff
@@ -2,54 +2,168 @@ package github_connection
 
 import (
 	"context"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
 	"github.com/google/go-github/v61/github"
```

Comment:
Code Review 2024-08-18 01:18:35 - New lines 1 to 5:

	"fmt"
	"os"
	"strconv"
	"strings"